react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,29 @@
+import './ToggleButton.css';
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('useState', isClicked);
+
+  const handleClick = () => {
+    console.log('before setter', isClicked);
+    if (isClicked === false) {
```

Comment:
let's try a different approach that looks cleaner. instead of using an if/else statement. 
there is a common way we write toggle functionality in programming